hazelcast/src/main/java/com/hazelcast/sql/impl/SqlDataSerializerHook.java

```diff
@@ -205,7 +206,9 @@
 
     public static final int EXPRESSION_TO_EPOCH_MILLIS = 74;
 
-    public static final int LEN = EXPRESSION_TO_EPOCH_MILLIS + 1;
+    public static final int MAP_INDEX_SCAN_METADATA = 75;
```

Comment:
I would keep suppliers & metadata serialization configuration together.

Reply:
Agree, but they are in different packages.

Reply:
Can't they be colocated?

Reply:
Yes, moved it to `hazelcast-sql` package.